proto/astarteplatform/msghub/node.proto

```diff
@@ -24,7 +24,7 @@ import "astarteplatform/msghub/interface.proto";
 
 package astarteplatform.msghub;
 
-/* This message defines a node to be attached/detached to the Astarte message hub. */
+/* This message defines a node to be attached to the Astarte message hub. */
 message Node {
   string uuid = 1;                      // The node identifier.
```

Comment:
mmm, I have a doubt, since we have the node-uuid at the metadata level, can it be removed from here as well?

Reply:
The attach phase is the only step in which we don't have the Node Id in the metadata field, so I think it's still necessary

Reply:
right, metadata is set at request level, not at the connection level.